free-courses-en.md

```diff
@@ -126,6 +126,11 @@
 ### Bootstrap
 
 * [Bootstrap 4 Quick Start: Code Modern Responsive Websites](https://www.udemy.com/share/101qsw/) - Brad Hussey (Udemy)
+* [Bootstrap tutorial for beginners](https://www.youtube.com/watch?v=0tv85fkdV14) - kudvenkat (YouTube)
```

Comment:
We prefer YouTube URLs to be to the `/playlist` endpoint rather than the `/watch` endpoint. Could you update this so users can get a better overview of the entire course?

Also, there's no need to specify YouTube explicitly.

```suggestion
* [Bootstrap tutorial for beginners](https://www.youtube.com/playlist?list=PL6n9fhu94yhXd4xnk-j5FGhHjUv1LsF0V) - kudvenkat
```